app/controllers/idv/review_controller.rb

```diff
@@ -124,6 +125,16 @@ def password
       params.fetch(:user, {})[:password].presence
     end
 
+    def confirm_verify_info_complete
+      # rubocop:disable Style/IfUnlessModifier
+      if IdentityConfig.store.doc_auth_verify_info_controller_enabled
+        if !idv_session.resolution_successful
+          redirect_to idv_verify_info_url
+        end
+      end
+      # rubocop:enable Style/IfUnlessModifier
```

Comment:
QQ what was the reason for disabling the lint? is this WIP?

Also, I think the lint would be quiet if we combined the two conditions to be one, so that it's too long to qualify for that modifier approach?

```suggestion
      if IdentityConfig.store.doc_auth_verify_info_controller_enabled && !idv_session.resolution_successful
        redirect_to idv_verify_info_url
      end
```

Reply:
Fixed in https://github.com/18F/identity-idp/pull/7661/commits/3a2c085d3310bc6a6be60b2f8f29acd7fe3037d0